src/js/components/EditsNav.jsx

```diff
@@ -153,17 +173,15 @@ export default class EditsNav extends Component {
     return (
       <div style={wrapperHeight}>
         <div className={`EditsNav ${fixedClass}`} id="editsNav">
-          <div className="nav-wrapper">
-            <ul className="usa-nav-primary">
-              {
-                navNames.map((pageObj, i) => {
-                  return renderLinkOrText(this.props, pageObj, i)
-                })
-              }
-            </ul>
-            <hr className="line" />
-            <hr className="progress" width={getProgressWidth(this.props)} />
-          </div>
+          <ul className="usa-nav-primary">
+            {
+              navNames.map((pageObj, i) => {
+                return renderLinkOrText(this.props, pageObj, i)
+              })
+            }
+          </ul>
+          <hr className="nav-bg" />
+          <hr />
```

Comment:
The _only_ benefit to the last `<hr />` is for when that goes sticky. It provides a break between the content and the nav in that case.